public/app/themes/justice/inc/security.php

```diff
@@ -69,4 +102,48 @@ public function setLoginPeriod(): float|int
     {
         return 7 * DAY_IN_SECONDS; // Cookies set to expire in 7 days.
     }
+
+    /**
+     * Change the URL of the script or style tags.
+     *
+     * @see https://developer.wordpress.org/reference/hooks/style_loader_tag/
+     *
+     * @param $tag string The HTML string of a link or script tag.
+     * @return string The modified HTML string.
+     */
+    public function filterAssetQueryString(string $tag): string
+    {
+        return str_replace('ver=' . $this->wp_version, 'ver=' . $this->hashed_wp_version, $tag);
+    }
+
+    /**
+     * Disable REST API for non-logged in users.
+     *
+     * @see https://developer.wordpress.org/reference/hooks/rest_authentication_errors/
+     *
+     * @param WP_Error|null|true $result
+     * @return WP_Error|null|true
+     */
```

Comment:
@EarthlingDavey I assume nothing is using the REST API that might break? 
I have a vague memory of there being something reliant on justice.gov.uk post-migration but it might just have been serving styles...

Maybe this one http://xhibit.justice.gov.uk/ ?

Reply:
Oh yeah, good shout. It was just static assets like js, css & images.